roles/letsencrypt/defaults/main.yml

```diff
@@ -16,13 +16,22 @@ letsencrypt_conf_dir_group: "{{ lookup('env', 'GROUP') }}"
 letsencrypt_prerequisites_packagemanager: "yum"
 # region for s3 bucket when
 letsencrypt_s3_config_region: "eu-west-1"
+# convert certificate to other format
+convert_certificate: false
 
 ### http challenge / dns challenge
 acme_directory: "https://acme-staging-v02.api.letsencrypt.org/directory"
+acme_live_directory: "https://acme-v02.api.letsencrypt.org/directory"
```

Comment:
The live directory is/was set via a fact at runtime when "letsencrypt_use_acme_live_directory: true". Why this one is needed?

Reply:
thats just the variabel definition for the set_fact
1. one place for variable defintion is better than over multiple files
2. in the actually setup you cant override the acme_live_directory variable

Reply:
If we have `acme_live_directory`, the other variable should be called `acme_staging_directory`.

Reply:
acme_directory is set over set_fact no need here